.github/workflows/build.yml

```diff
@@ -175,21 +175,21 @@ jobs:
     steps:
       - name: Setup | Developer Command Prompt (x86)
         if: ${{ matrix.bits == 32 }}
-        uses: ilammy/msvc-dev-cmd@v1
+        uses: ilammy/msvc-dev-cmd@674ff850cbd739c402260838fa45b7114f750570
```

Comment:
Why this revision? I presume there's no release yet for the new version. If so, can you file a follow up to move to a tagged version when it's there?

Reply:
Sorry, I should have been more specific. From my understanding, all of these actions can change without us noticing. This is okay for some trusted sources like actions/ but always bothered me for some other actions. By using an explicit commit we make sure that we fixate this code. This is described in more detail in the link I posted in #84.

Reply:
I've enabled auto-merge and re-requested a review. When you think this is fine please resolve this conversation to unlock the auto-merge.

Reply:
This will probably create more work than necessary. But it's fine for now.